tensorflow/core/platform/default/logging.h

```diff
@@ -40,6 +40,7 @@ class LogMessage : public std::basic_ostringstream<char> {
  public:
   LogMessage(const char* fname, int line, int severity);
   ~LogMessage();
+  static int min_lvl_;
```

Comment:
I would prefer to follow comments from @vrv, adding a method and use a static variable to make logic more consistent to normal LOG behavior.

Thoughts?